book/features/tui/copy.md

```
@@ -1 +1,38 @@
+to copy it to the clipboard. Alternatively, press the corresponding character of the entry to directly copy it.
+
+## Available Copy targets
+
+- (`c`) `Command line`: the reconstructed commandline
+- (`o`) `Command line with full env`: the reconstructed commandline with full set of environment variables.
+- *Experimental.* (`s`) `Command line with Stdio`: the reconstructed commandline with stdio file descriptors.
+- *Experimental.* (`f`) `Command line with File descriptors`: the reconstructed commandline with file descriptors.
+- (`e`) Environment Variables: environment variables in `"KEY"="VALUE"` format.
+- (`d`) Diff of environment variables: diff of environment variables in the following format.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)